management-app/src/api/tasksAPI.ts

```diff
@@ -0,0 +1,75 @@
+import { BaseAPI } from './baseAPI';
+import { TASKS_API_ERRORS } from './errors';
+import { TTask, TTaskBase } from './types';
+
+class TasksAPI extends BaseAPI {
+  createTask(
+    boardId: string,
+    columnId: string,
+    token: string,
+    task: TTaskBase
```

Comment:
If possible, avoid passing in more than two parameters. One of the options is to aggregate params into a plain object:
```suggestion
  createTask({ boardId, columnId, token, task} : {
    boardId: string,
    columnId: string,
    token: string,
    task: TTaskBase})
```

Reply:
Thanks for advise